src/Chapter1.hs

```diff
@@ -614,8 +627,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let ldig = abs (n `rem` 10)
+      newnum = abs (n `quot` 10)
```

Comment:
Perfect work on using the standard functions! 👍🏼 

On a side note, there is a standard function as well that combines these two: [`quotRem`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:quotRem).

It return the pair, so you can pattern match on that and get both vars at once:

```haskell
let (ldig, newnum) = (abs n) `quotRem` 10
```